### PR TITLE
Update 06_Skills.txt

### PR DESCRIPTION
**Pull request by rosebrallier**
Changed 5 to 6 re Issue #458 
Updated some typography

### CODE REVIEW

File: CharacterCreation/06_Skills.txt [hunk 1]
```diff
@@ -51,48 +51,48 @@ often. Talk to your GM if you would like your character to learn a more specific
 case of one of these skills or a skill not listed below.
 
 == Charisma ==
-Deception:
+Deception-
```
Comment (Turtlelord26) [inline CharacterCreation/06_Skills.txt]: "Deception:" will parse as a topic header, while "Deception-" won't parse as anything. What was wrong with the colons?

Comment (rosebrallier) [reply CharacterCreation/06_Skills.txt]: Right! I forgot that I used the colons on purpose.

**Review by Turtlelord26 — APPROVED** [review]
👍 Ship the thing!

**Review by bleehu — APPROVED** [review]
Shipit real good!

**Pull request merged**